media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -18,6 +18,7 @@ LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/oferkv/phototonic/archive/v$portVersion.tar.gz"
 CHECKSUM_SHA256="a8b2dbc81750efabb65a4732b1704641afd4a4f772ef9c90817871ca2a52f5ef"
+SOURCE_FILE="v$portVersion.tar.gz"
```

Comment:
Oops,
```diff
-SOURCE_FILE="v$portVersion.tar.gz"
+SOURCE_FILE="phototonic-$portVersion.tar.gz"
```

Reply:
but @fbrosson  file itself is v2.1.tar.gz in current recipe..

Reply:
We define a `SOURCE_FILENAME` to let haikuporter know that we want it to rename the archive to something more meaningful than just `v2.1.tar.gz`.
([HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name))

Reply:
Sorry, it's my fault. I had written `SOURCE_FILE` instead of `SOURCE_FILENAME`.
If you try with
```diff
-SOURCE_FILE="v$portVersion.tar.gz"
+SOURCE_FILENAME="phototonic-$portVersion.tar.gz"
```
it will work.